packages/driver/src/cypress/stack_utils.ts

```diff
@@ -142,13 +142,28 @@ const getCodeFrameFromSource = (sourceCode, { line, column: originalColumn, rela
     line,
     column,
     originalFile: relativeFile,
-    relativeFile,
+    relativeFile: getRelativePathFromRoot(relativeFile, absoluteFile),
     absoluteFile,
     frame,
     language: getLanguageFromExtension(relativeFile),
   }
 }
 
+const getRelativePathFromRoot = (relativeFile, absoluteFile) => {
+  // at this point relativeFile is relative to the cypress config
+  // we need it to be relative to the repo root, which is different for monorepos
+  const repoRoot = Cypress.config('repoRoot')
+  const posixAbsoluteFile = (Cypress.config('platform') === 'win32')
+    ? absoluteFile?.replaceAll('\\', '/')
```

Comment:
I wanted to use `path.posix` here per the [Writing cross-platform JavaScript](https://github.com/cypress-io/cypress/blob/develop/guides/writing-cross-platform-javascript.md) guide but it was undefined on path :(

Reply:
`path` is only available in Node.js so it won't show up in front-end code :+1:  

Looks good!

Reply:
Can sync up if you want to know more about this kind of thing (bundling, how it works etc), but I think what's happening is this: webpack will automatically polyfill node core modules in an attempt to let you bundle some code originally written for Node.js in the browser. 

Webpack 5 dropped this (since the idea of an auto polyfill is not good; developers should not be bundling Node.js code for the browser, doesn't really make sense).

Ref: https://gist.github.com/ef4/d2cf5672a93cf241fd47c020b9b3066a

Reply:
Ohhhhhhhhh mega interesting! Thank you for the explanation! 💟

Reply:
I think webpack is going away soon!! https://github.com/cypress-io/cypress/pull/23928